src/server/local/index.js

```diff
@@ -176,6 +176,21 @@ export default server => {
       });
       sendDataFile(res, dataFilePath, next);
     })
+    .get(
```

Comment:
Could we add a test for this to https://github.com/bbc/simorgh/blob/latest/src/server/local/index.test.js

Reply:
done